packages/firebase_analytics/firebase_analytics_web/lib/firebase_analytics_web.dart

```diff
@@ -67,6 +67,11 @@ class FirebaseAnalyticsWeb extends FirebaseAnalyticsPlatform {
   Future<void> setConsent({
     bool? adStorageConsentGranted,
     bool? analyticsStorageConsentGranted,
+    bool? adPersonalizationSignalsConsentGranted,
+    bool? adUserDataConsentGranted,
+    bool? functionalityStorageConsentGranted,
+    bool? personalizationStorageConsentGranted,
+    bool? securityStorageConsentGranted,
   }) async {
     throw UnimplementedError('setConsent() is not supported on Web.');
```

Comment:
Could you set up `setConsent()` API for web, please? [Here is the documentation](https://firebase.google.com/docs/reference/js/analytics.md#setconsent_1697027)

Reply:
Hope this is ok. 

https://github.com/firebase/flutterfire/pull/12298/commits/45f02c60582b79f5d21dc499844ca12927f0143e

I need to test it now ...